sysutils/dssystem/files/dssystem.in

```diff
@@ -31,6 +31,16 @@ dssystem_prestart()
     IP=`cat /root/dhcpip`
     sed -i.bak -E "s|codebase=\"(.+)\"|codebase=\"http://$IP/asigra/\"|" /usr/local/www/asigra/DSOP.jnlp
     rm -f /usr/local/www/asigra/DSOP.jnlp.bak
+
+    dssys_file="/etc/dssys.cfg"
+    # We make sure that the port is not added to the IP Address
+    ip=$(echo ${IP} | cut -d':' -f1)
+    grep -q "External IP Addr              :" "${dssys_file}"
```

Comment:
Do we really nned it to be this exact, with that many spaces?

Reply:
Well one space should work as well but all other entries in that file are padded that way - so I thought I could be consistent with those ?

Reply:
Its not a file that people see, i think we prefer it to be more bullet proof than consistent, specially because we dont control this file, it comes from Asigra.